plugin/trino-hive/src/main/java/io/trino/plugin/hive/util/HiveUtil.java

```diff
@@ -184,6 +185,15 @@
     public static final String ICEBERG_TABLE_TYPE_NAME = "table_type";
     public static final String ICEBERG_TABLE_TYPE_VALUE = "iceberg";
 
+    public static final String HUDI_TABLE_TYPE_NAME = "type";
+    public static final String HUDI_TABLE_TYPE_COPY_ON_WRITE = "cow";
+    public static final String HUDI_TABLE_TYPE_MERGE_ON_READ = "mor";
```

Comment:
Change to private and use `cow` and `mor` literals in the product test.

Reply:
done.